ports/rp2/boards/ADAFRUIT_FEATHER_RP2040/mpconfigboard.h

```diff
@@ -1,3 +1,19 @@
-// Board and hardware specific configuration
-#define MICROPY_HW_BOARD_NAME                   "Adafruit Feather RP2040"
-#define MICROPY_HW_FLASH_STORAGE_BYTES          (3072 * 1024)
+// https://www.adafruit.com/product/4884
+// https://learn.adafruit.com/adafruit-feather-rp2040-pico/pinouts
+
+#define MICROPY_HW_BOARD_NAME          "Adafruit Feather RP2040"
+#define MICROPY_HW_FLASH_STORAGE_BYTES (7 * 1024 * 1024)
+// Early revision green PCB boards shipped with 4MB flash instead of 8MB
+// If you are using one, change that 7 above to a 3
```

Comment:
So this board definition will no longer work on older 4MB boards.  I'm not sure how much that matters, were there many boards out there with 4MB?  What did CircuitPython do here, does it configure 4MB or 8MB or something else?

Reply:
From what I can tell, the 4MB version was the initial green pcb development boards, which have since switched to the 8MB (64Mbit) GD25Q64C chips. Not sure how many green boards are out in the wild.

CircuitPython is currently defining the 8MB chip:
https://github.com/adafruit/circuitpython/blob/main/ports/raspberrypi/boards/adafruit_feather_rp2040/mpconfigboard.mk#L9

Reply:
Raised the question in the Adafruit forum: https://forums.adafruit.com/viewtopic.php?f=60&t=181723

Reply:
Confirmed by Scott. It never shipped with 4MB.